src/isIsogram.test.js

```diff
@@ -6,4 +6,29 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
```

Comment:
not bad, but add 2-3 more tests with your own ideas

p.s. Pay attention that you did not provide any positive  tests where input is an isogram